STACpopulator/implementations/CORDEXCMIP6_Ouranos/add_CORDEX6.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from requests.sessions import Session
+
+from STACpopulator.requests import add_request_options, apply_request_options
+from STACpopulator.input import ErrorLoader, THREDDSLoader
+
+import argparse
+from typing import Any
+from STACpopulator.populator_base import STACpopulatorBase
+from STACpopulator.extensions.cordex6 import Cordex6DataModel
+
+LOGGER = logging.getLogger(__name__)
+
+
+class CORDEX_STAC_Populator(STACpopulatorBase):
+    data_model = Cordex6DataModel
+    item_geometry_model = None  # Unnecessary, but kept for consistency
+
+    def create_stac_item(self, item_name: str, item_data: dict[str, Any]) -> dict[str, Any]:
+        dm = self.data_model.from_data(item_data)
+        return dm.stac_item()
+
+
+
+# TODO: This probably doesn't need to be copied for every implementation, right ?
```

Comment:
If there's a lot of common arguments we can factor them out into their own function like we did for add_request_options and add_logging_options